src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
```

Comment:
I'm on board with that, so long as we don't use classes as quasi-IDs. Suggestions to replace? `body > footer` is moderately complicated for the rendering engines, but it's pretty clear.